hedyorg/hedy
# 🧪 Run slow test suites for all PRs with caching

- Add all hedy_*.py files to the source code cache generation.
- Enable slow test suites to run for all PRs, with caching.

Fixes #5573
**How to test**
- Ensure that unit tests job on GitHub runs all tests. There should be 2 runs: 1) ~8000 tests and 2) ~23000 tests. Note that the first run will be very slow. All consecutive runs should be quicker.
